items.json

```diff
@@ -49246,14 +49246,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_eastern_heavy_mace_v1_h0",
-    "baseId": "crpg_eastern_heavy_mace_v1",
+    "id": "crpg_eastern_heavy_mace_v2_h0",
+    "baseId": "crpg_eastern_heavy_mace_v2",
     "name": "Eastern Heavy Mace",
     "culture": "Khuzait",
     "type": "OneHandedWeapon",
-    "price": 3728,
-    "weight": 2.04,
-    "tier": 6.99636,
+    "price": 2948,
+    "weight": 2.12,
+    "tier": 6.09928036,
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.